plugin-enclave-gradle/src/main/kotlin/com/r3/conclave/plugin/enclave/gradle/GenerateEnclaveMetadata.kt

```diff
@@ -13,9 +15,20 @@ open class GenerateEnclaveMetadata @Inject constructor(
     private val buildType: BuildType,
     private val linuxExec: LinuxExec
 ) : ConclaveTask() {
+    companion object {
+        const val ENCLAVE_MRSIGNER_FILE = "mrsigner.txt"
+        const val ENCLAVE_MRENCLAVE_FILE = "mrenclave.txt"
+    }
```

Comment:
Minor: I don't see the need for these constants, since the location of the files are defined by the plugin class.

Reply:
Removed